io.github.yairm210.unciv.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: io.github.yairm210.unciv
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk11]
+command: unciv
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: [/usr/lib/sdk/openjdk11/install.sh]
+
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.1.tar.xz
+        sha256: 7bc76daf9d72f8aff885efad04ce06b90488a1a169d118dea8a2b661832e8762
+
+  - name: unciv
+    buildsystem: simple
+    build-commands:
+      - install -D unciv-wrapper.sh /app/bin/unciv
+      - install -D Unciv.jar /app/data/Unciv.jar
+      - install -Dm644 -t /app/share/metainfo io.github.yairm210.unciv.appdata.xml
+      - install -Dm644 -t /app/share/applications io.github.yairm210.unciv.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/512x512/apps io.github.yairm210.unciv.png
+    sources:
+      - type: file
+        path: unciv-wrapper.sh
+      - type: file
+        path: io.github.yairm210.unciv.desktop
+      - type: file
+        path: io.github.yairm210.unciv.png
+      - type: file
+        path: io.github.yairm210.unciv.appdata.xml
+      - type: file
```

Comment:
As mentioned in the main thread, you should also add source tarball here. It later gets exported to separate extension and can be inspected by users.

Reply:
Thanks for the reminder! I wrote it down on https://github.com/flathub/io.github.yairm210.unciv/issues/3